common/src/main/java/com/microsoft/identity/common/internal/cache/CacheKeyValueDelegate.java

```diff
@@ -192,6 +196,9 @@ public String generateCacheKey(Credential credential) {
             final IdTokenRecord idToken = (IdTokenRecord) credential;
             cacheKey = cacheKey.replace(REALM, sanitizeNull(idToken.getRealm()));
             cacheKey = cacheKey.replace(TARGET, "");
+        } else if (credential instanceof PrimaryRefreshTokenRecord) {
+            cacheKey = cacheKey.replace(REALM, "");
+            cacheKey = cacheKey.replace(TARGET, "");
```

Comment:
Same here.... I don't understand this. Why is this value being set to `""`? Can you point me towards where this is outlined in the schema spec?

Reply:
OK - I think is same as [here](https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=%2FSSO%2FSchema.md&version=GBiambmelt%2Fclient_server_nonce&line=242&lineEnd=243&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents), please see [this comment](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1177/files#r562968385) re: updating filter logic used for serialized instances